CMakeLists.txt

```diff
@@ -495,6 +492,12 @@ if(WITH_RADOSGW)
   endif()  # OPENSSL_FOUND
 endif (WITH_RADOSGW)
 
+#option for ceph-mds
+option(WITH_MDS "build ceph-mds daemon" ON)
+
+# building libcephfs needed for client library
+option(WITH_LIBCEPHFS "build libcephfs client library" ON)
```

Comment:
WITH_LIBCEPHFS is used above around line 375: if(WITH_LIBCEPHFS OR WITH_KRBD)

Reply:
@cbodley 
Note that on FreeBSD is build with both `WITH_LIBCEPFS` and `WITH_CEPHFS` OFF

the part you quottaed might have to do with both `mount.ceph` and `krbd` using keyutils to store secrets. 
```
./src/mount/mount.ceph.c:8:#include "common/secret.h"
./src/krbd.cc:34:#include "common/secret.h"
```
So perhaps there it should be changed to `if(WITH_CEPHFS OR WITH_KRBD)`
But then building `mount` on src/CMakeLists.txt:623 and on already puts it under `WITH_LIBCEPHFS` since 2014, so I guess I just followed suite.

So I guess there is some inconsistent usage of `WITH_LIBCEPHFS`??